beacon-chain/node/config.go

```diff
@@ -117,7 +118,15 @@ func configureExecutionSetting(cliCtx *cli.Context) error {
 	if !common.IsHexAddress(ha) {
 		return fmt.Errorf("%s is not a valid fee recipient address", ha)
 	}
-	c.DefaultFeeRecipient = common.HexToAddress(ha)
+	mixedcaseAddress, err := common.NewMixedcaseAddressFromString(ha)
+	if err != nil {
+		return errors.Wrapf(err, "could not decode fee recipient %s", ha)
+	}
+	checksumAddress := common.HexToAddress(ha)
+	if !mixedcaseAddress.ValidChecksum() {
+		log.Warnf("fee recipient %s is not a valid checksum address. The checksummed address is %s and this will be used as the fee recipient", ha, checksumAddress.Hex())
```

Comment:
Let's consider this log from the user's perspective. Many do not know what a checksum is, so we should instead warn them that a checksummed address is safer to prevent against spelling mistakes in Ethereum addresses. The provided address was not checksummed and will be used as is when setting transaction fee recipients

Reply:
i'll go ahead and update this

Reply:
Maybe something about how we recommend using a mixed-case address to prevent against spelling mistakes in your fee recipient Ethereum address. The provided address is not a valid checksum (mixed-case) address. Not sure if we should link people to a specific place, but something worth thinking about